### DISCUSSION TITLE
Overlapping Power commands in PulseTime mode

### DISCUSSION THREAD

**Discussion by KWolfe81** [arendst/Tasmota#16259] [Q&A] [answered]
Hiyas,

(Preface: sorry for the n00b question, I'm in the planning stages and haven't even loaded softawre on a single device yet.)

Do new commands automatically override existing commands?  As in, say I set PulseTime to turn the device on for 2 seconds, and then issue 5x Power commands, one second apart, with each command instructing Relay0 to close for 2 seconds.  Will I get Relay0 to stay closed, uninterrupted, for exactly 6 seconds?  Or do commands queue up (relay stays closed for 10 seconds, and may or may not be continuous)?

EDIT: Trying to answer my own question - looks like it'll be on for 6 seconds as the Power command documentation states:  "also restarts PulseTime"

**Comment by sfromis** [comment, marked as answer]
Commands do not queue up or override (whatever that would mean in each case), but each will simply act at the time of execution, including the Power commands command setting a new PulseTime expiry time. This is a good fit for, say, a PIR where it makes sense to keep the light on until there is no more activity.

**Comment by KWolfe81** [comment]
Perfect, exactly what I need. Thanks!
